src/intrprtr.c

```diff
@@ -2587,10 +2614,10 @@ void            IntrRecExprBegin (
     /* if this is an outmost record, save it for reference in '~'          */
     /* (and save the old value of '~' on the values stack)                 */
     if ( top ) {
-        old = VAL_GVAR( Tilde );
+        old = STATE( Tilde );
         if ( old != 0 ) { PushObj( old ); }
         else            { PushVoidObj();  }
-        AssGVarUnsafe( Tilde, record );
+        STATE(Tilde) = record;
```

Comment:
Here and in all these other places, we have `STATE(Tilde)` versus `STATE( Tilde )` (three lines above).

Don't get me wrong, I won't hold this PR hostage over spaces... :-).